pyramid_chameleon/precompile.py

```diff
@@ -103,28 +115,41 @@ def precompile(argv=sys.argv):
             default=1)
     options, args = parser.parse_args(argv)
```

Comment:
Seems like it would be logical to have a CLI option for `cache_dir`, perhaps defaulting to the value of `chameleon.config.cache_dir`?

Reply:
I can imagine someone wanting to run this at Pyramid startup time, so I'll left the CHAMELEON_CACHE way to spell the cache directory in but added a ``--cache-dir`` option.